client/src/App.js

```diff
@@ -12,44 +12,165 @@ import Login from "./pages/Login";
+  "redSpy": "1",
+  "redGuessers": ["4"],
+  "blueSpy": "BBonnie",
+  "blueGuessers": ["3"],
+  "isReady": true,
+  "isStart": true,
+  "cards": {
+      "banyan": "grey",
+      "alb": "grey",
+      "yarn": "blue",
+      "chop": "grey",
+      "alpaca": "blue",
+      "jail": "blue",
+      "gauge": "red",
+      "marble": "red",
+      "hedge": "red",
+      "scrutiny": "red",
+      "botany": "grey",
+      "CD": "red",
+      "croup": "blue",
+      "brook": "grey",
+      "steamroller": "blue",
+      "cutting": "grey",
+      "designation": "blue",
+      "stew": "blue",
+      "merchandise": "grey",
+      "adrenalin": "grey",
+      "output": "blue",
+      "chrome": "black",
+      "stacking": "red",
+      "meaning": "red",
+      "origin": "red"
+  },
+  "blueCardNum": 8,
+  "redCardNum": 8,
+  "whiteCardNum": 8,
+  "blackCardNum": 1,
+  "redPoints": 0,
+  "bluePoints": 0,
+  "turn": "blue",
+  "guessNum": 0,
+  "isEnd": false,
+  "boardState": {
+      "banyan": {
+        "status": "covered"
+      },
+      "alb": {
+        "status": "covered"
+      },
+      "yarn": {
+        "status": "covered"
+      },
+      "chop": {
+        "status": "covered"
+      },
+      "alpaca": {
+        "status": "covered"
+      },
+      "jail": {
+        "status": "covered"
+      },
+     "gauge": {
+        "status": "covered"
+      },
+      "marble": {
+        "status": "covered"
+      },
+      "hedge": {
+        "status": "covered"
+      },
+      "scrutiny": {
+        "status": "covered"
+      },
+      "botany": {
+        "status": "covered"
+      },
+      "CD": {
+        "status": "covered"
+      },
+      "croup": {
+        "status": "covered"
+      },
+      "brook": {
+        "status": "covered"
+      },
+      "steamroller": {
+        "status": "covered"
+      },
+      "cutting": {
+        "status": "covered"
+      },
+      "designation": {
+        "status": "covered"
+      },
+      "stew": {
+        "status": "covered"
+      },
+      "merchandise": {
+        "status": "covered"
+      },
+      "adrenalin": {
+        "status": "covered"
+      },
+      "output": {
+        "status": "covered"
+      },
+      "chrome": {
+        "status": "covered"
+      },
+      "stacking": {
+        "status": "covered"
+      },
+      "meaning": {
+        "status": "covered"
+      },
+      "origin": {
+        "status": "covered"
+      }
+  },
+  "timer": 10
+};
+
 let socket = socketIOClient();
 function App() {
-  const [state, setState] = useState({player: "Bonnie", gameState: undefined});
-  const [gameID, setGameID] = useState(undefined);
+  const [gameID, setGameID] = useState(undefined),
+    [state, setState] = useState({player: "Bonnie", gameState: test}),
+    withGameState = Component => props => (
+      <Component
+        state={state}
+        setState={setState}
+        gameID={gameID}
+        setGameID={setGameID}
+        socket={socket}
+        {...props}
+      />
+    );
```

Comment:
According to https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards, "Each variable assignment SHOULD be declared on a separate line - including variables that are only declared but do not get a value assigned."

Reply:
Can you clarify what this would look like according to the coding standards? I think this qualifies as being on a separate line than the other variables.